.github/workflows/update_app_templates_hash.yml

```diff
@@ -0,0 +1,43 @@
+name: Update aws/aws-sam-cli with latest commit hash from aws/aws-sam-cli-app-templates
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  workflow_dispatch: {}
+
+jobs:
+  updateCommitHash:
+    permissions:
+      pull-requests: write
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout App Templates
+        uses: actions/checkout@v3
+        with:
+          repository: aws/aws-sam-cli-app-templates
+          path: aws-sam-cli-app-templates
+
+      - name: Checkout SAM CLI
+        uses: actions/checkout@v3
+        with:
+          repository: aws/aws-sam-cli
+          path: aws-sam-cli
+
+      - name: Update commit hash in SAM CLI from App Templates
+        run: |
+          cd aws-sam-cli-app-templates
+          export APP_TEMPLATES_COMMIT_HASH=$(git rev-parse HEAD)
+          cd ../aws-sam-cli
+          cat <<< $(jq --arg commit_hash "$APP_TEMPLATES_COMMIT_HASH" --indent 2 '.app_template_repo_commit =  $commit_hash' samcli/runtime_config.json) > samcli/runtime_config.json
+
+      - name: Raise PR for SAM CLI
```

Comment:
If the previously PR is not merged, will this close the previous PR? 
The reason for asking is that if we merge the latest PR (with the latest app template commit hash), and leave the older PR(s) open, there could be risk that we would accidentally merge them afterwards.

Reply:
No this will update the existing PR. And if the branch is out of date (like there are new commits in develop) this will update with them as well.